src/amo/components/AddonReview.js

```diff
@@ -83,25 +83,38 @@ export class AddonReviewBase extends React.Component {
       );
     }
 
+    let placeholder;
+    let prompt;
+    if (review.rating && (review.rating > 3)) {
+      prompt = i18n.gettext(
+        'Tell the world why you think this extension is fantastic!'
+      );
+      placeholder = i18n.gettext(
+        'Tell us what you love about this extension. Be specific and concise.'
+      );
+    } else {
+      prompt = i18n.gettext(
+        'Tell the world about this extension.'
+      );
+      placeholder = i18n.gettext(
+        'Tell us about your experience with this extension. ' +
```

Comment:
We should see about using the linter's `singleLineString` here (https://muffinresearch.co.uk/removing-leading-whitespace-in-es6-template-strings/)